.travis.yml

```diff
@@ -10,10 +15,12 @@ php:
 matrix:
     allow_failures:
         - php: hhvm
+    fast_finish: true
 
 before_script:
-  - ./bin/configure_test_env.sh
-  - composer self-update && composer install --dev --prefer-source --no-interaction
+  - echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - cp tests/Gaufrette/Functional/adapters/DoctrineDbal.php.dist tests/Gaufrette/Functional/adapters/DoctrineDbal.php -f
+  - composer self-update && composer install --dev --prefer-source --no-interaction --ignore-platform-reqs

```

Comment:
``--dev``is a deprecated no-op option (well, actually, it does something: displaying a deprecation warning)